data/wanderer/wanderers start.txt

```diff
@@ -179,67 +183,388 @@ mission "Wanderers: Hai Diplomat"
+			`	"Captain <last>," squeaks a Hai voice near you. "I'm Eenap, Professor Andelena's grad student. Please follow me to her office so we may discuss an expedition." Soon, you arrive at Professor Andelena's office, which looks like a small museum exhibit, filled with taxidermy statues of evolutionary ancestors of the Hai, and diagrams of development of Hai society.`
+			`	"Welcome, Captain <last>! I am Professor Maindell Stormchild Andelena, IV." This human woman speaks with the famously-dignified Shroud cloud city accent. "We were planning an expedition to study planets suspected of being part of a past Hai empire, but your discovery has piqued our interest."`
 			choice
-				`	"Indeed, I'm hoping you have someone in mind."`
-				`	"It would be great to work with you again if you're willing."`
-			`	You open your mouth to respond but the effort is pointless.`
-			`	"I accept!" she cries, leaning forward on the desk with a gleeful look in her eyes.`
-			`	"A first contact is a dream opportunity, and there are so very few of them. So of course I will come with you."`
+				`	"You said you were... Skiorologists?"`
+				`	"My only interest is translation. Will you do that for me?"`
+					goto "get to the point"
+			`	"As Skiorologists, we study the development of the Hai, in the same way that Anthropologists study humans. How the Hai interact with other species is a major part of what makes the Hai... Hai. We have researched past interactions, but analyzing an ongoing interaction is a dream come true.`
+			label "get to the point"
+			`	"In return for introducing us to this new species, we'll be happy to translate for you. As we're already prepared to leave for an expedition, we can leave as soon as you meet us in the spaceport."`
 			choice
-				`	"Let's go visit these aliens immediately!"`
+				`	"Sure! I'll meet you at the spaceport."`
 					accept
-				`	"Can a distinguished ambassador such as yourself just leave like that?"`
-				`	"What if the humans want our own envoy to these aliens?"`
-					goto envoy
-			`	"Yes," she says. "If these aliens will provide a place for me, I mean to live with them permanently, an ambassador between their people and ours. I am old, and in need of a great novelty like this to make my life interesting once more." You note that the fur on her face, especially on the snout, has turned from brown to silvery gray - a sign of age in terrestrial creatures, and perhaps in the Hai as well.`
-			`	"Besides," she says, "after recent events they can hardly say no to me."`
+				`	"No, I'd rather wait for the Hai government to provide an ambassador."`
+					decline
+	on accept
+		log "Professor Andelena and her grad student agreed to act as interpreters for the aliens to the north. I need to meet them at the <planet> spaceport."
```

Comment:
Does <planet> work like this?

Reply:
I don't know what you're asking.

Reply:
Will `<planet>` work in this way?

Reply:
The `<planet>` will automatically be replaced with the destination planet name.

Reply:
Can be resolved than.